source/whats-new.txt

```diff
@@ -4,20 +4,30 @@
 What's New
 ==========
 
-.. default-domain:: mongodb
-
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
```

Comment:
now that there are 2 releases, should we include a list at the top for quick navigation? such as the [Node page](https://www.mongodb.com/docs/drivers/node/current/whats-new/) has?